packages/shopware-6-client/src/services/pluginService.ts

```diff
@@ -31,3 +31,16 @@ export function invokeGet(
 ): Promise<any> {
   return contextInstance.invoke.get(address);
 }
+
+/**
+ * Invoke custom PATCH request to shopware API. Mostly for plugins usage.
+ * You can skip domain and pass only endpoint ex. `/api/my/endpoint`
+ * @throws ClientApiError
+ * @beta
+ */
+export function invokePatch(
+  { address }: { address: string },
+  contextInstance: ShopwareApiInstance = defaultInstance
+): Promise<any> {
+  return contextInstance.invoke.patch(address);
```

Comment:
Please handle the `payload` parameter as well - not only the address - as you predicted in your tests here:
```
address: "/some/patch/endpoint",
payload: {
   some: "payload",
},
```

Reply:
I also noticed it :), working on it, I was a bit roo fast with my pr :)